portfolio/src/main/webapp/script.js

```diff
@@ -178,4 +178,24 @@ function visualizeMap() {
   map.setMapTypeId('styles_map');
   homeInfoWindow.open(map, markerHome);
   lakeInfoWindow.open(map, markerLake);
+
+  favoritePlaces();
```

Comment:
A more descriptive function name like `loadFavoritePlaces` would be easier to understand at a glance